src/python/pants/core/goals/export.py

```diff
@@ -51,7 +52,7 @@ class Symlink:
 
 @frozen_after_init
 @dataclass(unsafe_hash=True)
-class ExportableData:
+class ExportResult:
```

Comment:
Why the renaming? It makes this harder to review, and it doesn't seem to achieve any purpose beyond preference. If anything, the new naming is less coherent, since "ExportRequest" sounds like a request to perform exporting, whereas it really is a request for data that the caller then actually exports.

Reply:
Because `Data` is awkward with plural. I didn't want to say `Datas`, nor `Datum`.